proxy/envoy/config.go

```diff
@@ -322,6 +324,20 @@ func buildOutboundRoutes(instances []*model.ServiceInstance, services []*model.S
 	return httpConfigs, tcpConfigs
 }
 
+func buildSslContext(hostname string, context *ProxyContext) *SslContext {
```

Comment:
nit: `buildSSLContext` to be consistent with the naming scheme (e.g. `buildTCPRoute` etc)?